README.md

```diff
@@ -12,9 +13,46 @@
 
 A community repository for Ansible Playbook of OpenSearch Project.
 
+## Single node OpenSearch Installation with Dashboards
+
+This ansible playbook supports the following,
+
+- Can be deployed on baremetal and VMs(AWS EC2)
+- Supports most popular **Linux distributions**(Centos7, RHEL7)
+- Install and configure the Apache2.0 opensource OpenSearch
+- Configure TLS/SSL for OpenSearch transport layer(Nodes to Nodes communication) and REST API layer
+- Generate self-signed certificates to configure TLS/SSL for opensearch
+- Configure the Internal Users Database with limited users and user-defined passwords
+- Install and configure the Apache2.0 opensource OpenSearch Dashboards
+
+### Prerequisite
+
+- **Ansible**
+- **Java 8**
+
+### Configure
+
+Refer the file `inventories/opensearch/group_vars/all/all.yml` to change the default values.
+
+For example if we need to increase the java memory heap size for opensearch,
+
+    xms_value: 8
+    xmx_value: 8
+
+
+### Install
+
+
+    # Deploy with ansible playbook - run the playbook as root
+    ansible-playbook -i inventories/opensearch/hosts opensearch.yml --extra-vars "admin_password=Test@123 kibanaserver_password=Test@6789"
+
+You should set the reserved users(`admin` and `kibanaserver`) password using `admin_password` and `kibanaserver_password` variables.
+
+It will install and configure the opensearch. Once the deployment completed, you can access the opensearch Dashboards with user `admin` and password which you provided for variable `admin_password`.
```

Comment:
Not consistent, is it `kibanaserver_password` or `admin_password`.

Reply:
@peterzhuamazon  We require both. admin password is required for users to proceed further (manage dashboard). kibanaserver password is required for Dashboard installation where its used in configuration file. 
Refer: https://opensearch.org/docs/dashboards/install/tls/